prow/cluster/upgrade.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+split() {
+  project=
+  zone=
+  cluster=${1##*/} # Remove everything before last /
+  if [[ "$cluster" == "$1" ]]; then
+    return 0
+  fi
+  pz=${1%/*} # Remove last / and everything after
+  project=${pz%%/*} # Everything before the first /
+  zone=${pz##*/} # Everything after the /
+}
+
+versions() {
+  out=( $(gcloud container get-server-config "--project=$project" "--zone=$zone" --format='value(validMasterVersions[0],validNodeVersions[0])') )
```

Comment:
```bash
Line 34:
  out=( $(gcloud container get-server-config "--project=$project" "--zone=$zone" --format='value(validMasterVersions[0],validNodeVersions[0])') )
        ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```

Reply:
Done